certs/providerplus.emishealthinsights.co.uk/get_certs.sh

```diff
@@ -0,0 +1,3 @@
+openssl s_client -showcerts -connect providerplus.emishealthinsights.co.uk:443 2>/dev/null </dev/null | \
```

Comment:
Could you stick a shebang at the top of this file?  IIRC it'll run in whatever shell you're calling it from otherwise.

We probably want `set -u` (fail a pipeline if any command fails) too.